addons/web-support/api/src/main/java/org/jboss/windup/web/addons/websupport/rest/graph/applicationDetails/HintReducedDTO.java

```diff
@@ -0,0 +1,60 @@
+package org.jboss.windup.web.addons.websupport.rest.graph.applicationDetails;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.enterprise.inject.Vetoed;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author <a href="mailto:jesse.sightler@gmail.com">Jesse Sightler</a>
+ */
+@Vetoed
+@JsonIgnoreProperties({"handler", "delegate"})
+public class HintReducedDTO
+{
+    private int title;
+    private int javaFQCN;
```

Comment:
What exactly is FQCN? I would assume it to be _fully qualified class name_, but since it is int variable, it is probably not.